_getting-started/concepts-system-overview-guide.md

```diff
@@ -0,0 +1,394 @@
+
+          The Extraction phase includes:
+
+          {% for sub-subsection in subsection.sub-subsections %}
+          - [{{ sub-subsection.title }}](#{{ sub-subsection.anchor }})
+          {% endfor %}
+
+        sub-subsections:
+          - title: "Step 1: The Singer-based replication engine"
+            anchor: "system-architecture--singer-based-rep-engine"
+            content: |
+              For SaaS applications and databases, data is pulled on a schedule using the Singer-based replication engine that Stitch runs against data sources like APIs, databases, and flat files.
+
+              During this step, two things happen:
+
+              1. **A structure sync**. At the start of every extraction, Stitch will run what is called a structure sync. A structure sync detects the tables and columns available in the source, along with any changes to the structure of those tables and columns.
+
+                 **Note**: New tables and columns detected during a structure sync will be available in Stitch after the current replication job ends. For the majority of integrations, new tables and columns won't be automatically set to replicate.
+              2. **Data is extracted in JSON format.** Based on the schemas, tables, and columns set to replicate, Stitch extracts the data in JSON format and sends it to the Import API.
+
+          - title: "Step 2: The Import API"
+            anchor: "system-architecture--import-api"
+            content: |
+              The next step in the replication process is the Import API. 
+
+              For data sent directly to Stitch through a webhook or Import API integration, this is the first step in the replication process. (**Note**: This is why webhook and Import API integrations don't have [Extraction Logs]({{ link.replication.extraction-logs | prepend: site.baseurl }}).)
+
+              The [Import API]({{ link.import-api.getting-started | prepend: site.baseurl }}) is a [Clojure](https://clojure.org/){:target="new"} web service that accepts JSON and [Transit](https://github.com/cognitect/transit-format){:target="new"}, either in large batches or at a point in time. The Import API performs a validation check on the data before writing it to a central Apache Kafka queue.
+
+              If the data fails validation or another critical error occurs, the extraction will fail and trigger an [in-app and email notification]({{ link.account.notification-settings | prepend: site.baseurl }}). The error can also be viewed in an integration's [Extraction Logs]({{ link.replication.extraction-logs | prepend: site.baseurl }}) tab.
+
+              ---
+
+      - title: "Preparing"
+        anchor: "system-architecture--preparing"
+        content: |
+          The second phase in the replication process is called **Preparing**. During this phase, the extracted data is replicated internally for redundancy, batched, and readied for loading.
```

Comment:
change "replicated internally for redundancy" to "buffered in Stitch's durable and redundant internal data pipeline"